packages/extensions-sdk/src/cli/commands/validate.ts

```diff
@@ -0,0 +1,102 @@
+import { EXTENSION_PKG_KEY, EXTENSION_TYPES } from '@directus/extensions';
+import chalk from 'chalk';
+import fse from 'fs-extra';
+import ora from 'ora';
+import { log } from '../utils/logger.js';
+
+export default async function validate(): Promise<void> {
+	const spinner = ora(chalk.bold('Validating Directus extension...')).start();
+
+	try {
+		await Promise.all([checkBuiltCode(), checkReadMe(), checkLicense(), checkDirectusConfig()]);
+
+		spinner.succeed(chalk.bold('Extension is valid'));
+	} catch (error) {
+		spinner.fail(chalk.bold('Failed validation'));
+		log(String(error), 'error');
+	}
+}
+
+async function checkBuiltCode() {
+	const spinner = ora('Check for /dist').start();
+
+	if (!(await fse.pathExists(`${process.cwd()}/dist`))) {
+		spinner.fail();
```

Comment:
Is this necessary given that it will output `spinner.fail` in L14 above for any error thrown?

Reply:
I wanted to make sure it was clear if multiple checks fail. If I didn't use individual instances of `spinner` it would only report one

Reply:
Interesting, in that case might be worth to use [Promise.allSettled](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled) instead of `Promise.all`. This way we can build a final error message to output based on output of all calls without the need for spinner fail.